### PR TITLE
Modify default pages with the new f5 iframe page loader

### PR DESCRIPTION
**Pull request by bwamie**
Modify default pages with the new f5 iframe page loader

### CODE REVIEW

File: csharp_dotnetcore/81.Facebook-Events/wwwroot/default.htm [hunk 1]
```diff
@@ -1,25 +1,419 @@
 ﻿<!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-    <title></title>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Facebook Bot</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/81.Facebook-Events/wwwroot/default.htm]: Consider changing this to Facebook Events bot. This is not the Facebook bot!

File: csharp_dotnetcore/23.LUIS-with-AppInsights/wwwroot/default.htm [hunk 2]
```diff
@@ -1,25 +1,419 @@
 ﻿<!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-    <title></title>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>LUIS Bot</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/23.LUIS-with-AppInsights/wwwroot/default.htm]: LUIS with Azure Application Insights

File: csharp_dotnetcore/22.QnA-with-AppInsights/wwwroot/default.htm [hunk 3]
```diff
@@ -1,25 +1,419 @@
 ﻿<!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-    <title></title>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>QnAMaker Bot</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/22.QnA-with-AppInsights/wwwroot/default.htm]: QnA Maker with Azure Application Insights

File: csharp_dotnetcore/21.Custom-Dialogs/wwwroot/default.htm [hunk 4]
```diff
@@ -1,25 +1,419 @@
 ﻿<!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-    <title></title>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>CustomDialogs Sample</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/21.Custom-Dialogs/wwwroot/default.htm]: Custom dialogs sample

File: csharp_dotnetcore/19.Bot-Authentication/wwwroot/default.htm [hunk 5]
```diff
@@ -1,24 +1,419 @@
...
-    <p>Describe your bot here and your terms of use etc.</p>
-    <p>To debug you bot using the Bot Framework Emulator, paste this URL into the emulator window:</p>
-    <div style="padding-left: 50px;" id="botBaseUrl"></div>
-    <p>Visit <a href="https://www.botframework.com/">Bot Framework</a> to register your bot. When you register, remember to set the endpoint to:</p>
-    <div style="padding-left: 50px;">
-        https://<strong>{your domain name}</strong>/api/messages
-    </div>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Authentication Bot</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/19.Bot-Authentication/wwwroot/default.htm]: Bot authentication sample

File: csharp_dotnetcore/18.Multi-Lingual-Bot/wwwroot/default.htm [hunk 6]
```diff
@@ -1,25 +1,419 @@
 ﻿<!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-    <title></title>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Authentication Bot</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/18.Multi-Lingual-Bot/wwwroot/default.htm]: Multi lingual bot

File: csharp_dotnetcore/16.Handling-Attachments/wwwroot/default.htm [hunk 7]
```diff
@@ -1,25 +1,419 @@
 ﻿<!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-    <title></title>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>HandlingAttachments Bot</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/16.Handling-Attachments/wwwroot/default.htm]: Handling attachments sample

File: csharp_dotnetcore/13.Basic-Bot-Template/wwwroot/default.htm [hunk 8]
```diff
@@ -1,24 +1,419 @@
...
-    <p>Describe your bot here and your terms of use etc.</p>
-    <p>To debug you bot using the Bot Framework Emulator, paste this URL into the emulator window:</p>
-    <div style="padding-left: 50px;" id="botBaseUrl"></div>
-    <p>Visit <a href="https://www.botframework.com/">Bot Framework</a> to register your bot. When you register, remember to set the endpoint to:</p>
-    <div style="padding-left: 50px;">
-        https://<strong>{your domain name}</strong>/api/messages
-    </div>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Basic Bot Sample</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/13.Basic-Bot-Template/wwwroot/default.htm]: Basic bot sample

File: csharp_dotnetcore/12.NLP-With-LUIS/wwwroot/default.htm [hunk 9]
```diff
@@ -1,24 +1,419 @@
...
-    <p>Describe your bot here and your terms of use etc.</p>
-    <p>To debug you bot using the Bot Framework Emulator, paste this URL into the emulator window:</p>
-    <div style="padding-left: 50px;" id="botBaseUrl"></div>
-    <p>Visit <a href="https://www.botframework.com/">Bot Framework</a> to register your bot. When you register, remember to set the endpoint to:</p>
-    <div style="padding-left: 50px;">
-        https://<strong>{your domain name}</strong>/api/messages
-    </div>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>LUIS Bot</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/12.NLP-With-LUIS/wwwroot/default.htm]: Natural language processing with LUIS.ai

File: csharp_dotnetcore/10.Prompt-Validations/wwwroot/default.htm [hunk 10]
```diff
@@ -1,25 +1,419 @@
 ﻿<!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-    <title></title>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>PromptValidations</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/10.Prompt-Validations/wwwroot/default.htm]: Prompt validations sample

File: csharp_dotnetcore/09.Message-Routing/wwwroot/default.htm [hunk 11]
```diff
@@ -1,25 +1,418 @@
 ﻿<!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-    <title></title>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (vishwacsena) [inline csharp_dotnetcore/09.Message-Routing/wwwroot/default.htm]: Missing title ..

File: csharp_dotnetcore/08.Suggested-Actions/wwwroot/default.htm [hunk 12]
```diff
@@ -1,25 +1,419 @@
 ﻿<!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-    <title></title>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>SuggestedActionsBot Sample</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/08.Suggested-Actions/wwwroot/default.htm]: Suggested actions sample

File: csharp_dotnetcore/07.Using-Adaptive-Cards/wwwroot/default.htm [hunk 13]
```diff
@@ -1,25 +1,419 @@
 ﻿<!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-    <title></title>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>UsingAdaptiveCardsBot Sample</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/07.Using-Adaptive-Cards/wwwroot/default.htm]: Using adaptive cards

File: csharp_dotnetcore/06.Using-Cards/wwwroot/default.htm [hunk 14]
```diff
@@ -1,25 +1,419 @@
 ﻿<!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-    <title></title>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>CardsBot Sample</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/06.Using-Cards/wwwroot/default.htm]: Using cards sample

File: csharp_dotnetcore/05.MultiTurn-Prompts-Bot/wwwroot/default.htm [hunk 15]
```diff
@@ -1,24 +1,419 @@
...
-    <p>Describe your bot here and your terms of use etc.</p>
-    <p>To debug you bot using the Bot Framework Emulator, paste this URL into the emulator window:</p>
-    <div style="padding-left: 50px;" id="botBaseUrl"></div>
-    <p>Visit <a href="https://www.botframework.com/">Bot Framework</a> to register your bot. When you register, remember to set the endpoint to:</p>
-    <div style="padding-left: 50px;">
-        https://<strong>{your domain name}</strong>/api/messages
-    </div>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>MultiTurnPromptsBot Sample</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/05.MultiTurn-Prompts-Bot/wwwroot/default.htm]: Multi turn prompts sample

File: csharp_dotnetcore/04.Simple-Prompt-Bot/wwwroot/default.htm [hunk 16]
```diff
@@ -1,25 +1,419 @@
 ﻿<!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-    <title></title>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>SimplePromptBot Sample</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/04.Simple-Prompt-Bot/wwwroot/default.htm]: Simple prompt sample

File: csharp_dotnetcore/03.Welcome-Users/wwwroot/default.htm [hunk 17]
```diff
@@ -1,24 +1,419 @@
...
-    <p>Describe your bot here and your terms of use etc.</p>
-    <p>To debug you bot using the Bot Framework Emulator, paste this URL into the emulator window:</p>
-    <div style="padding-left: 50px;" id="botBaseUrl"></div>
-    <p>Visit <a href="https://www.botframework.com/">Bot Framework</a> to register your bot. When you register, remember to set the endpoint to:</p>
-    <div style="padding-left: 50px;">
-        https://<strong>{your domain name}</strong>/api/messages
-    </div>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Welcome User Bot Sample</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/03.Welcome-Users/wwwroot/default.htm]: Welcome user sample

File: csharp_dotnetcore/02.EchoBot-With-Counter/wwwroot/default.htm [hunk 18]
```diff
@@ -1,24 +1,419 @@
...
-    <p>Describe your bot here and your terms of use etc.</p>
-    <p>To debug you bot using the Bot Framework Emulator, paste this URL into the Emulator window:</p>
-    <div style="padding-left: 50px;" id="botBaseUrl"></div>
-    <p>Visit <a href="https://www.botframework.com/">Bot Framework</a> to register your bot. When you register, remember to set the endpoint to:</p>
-    <div style="padding-left: 50px;">
-        https://<strong>{your domain name}</strong>/api/messages
-    </div>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Echo with Counter Bot</title>
```
Comment (vishwacsena) [inline csharp_dotnetcore/02.EchoBot-With-Counter/wwwroot/default.htm]: Echo bot with counter sample

**Comment by JasonSowers** [comment]
@vishwacsena can we merge now?

**Pull request merged**